python/paddle/fluid/layers/nn.py

```diff
@@ -2349,6 +2350,12 @@ def batch_norm(input,
         moving_variance_name(string, Default None): The name of the moving_variance which store the global Variance.
         do_model_average_for_mean_and_var(bool, Default False): Do model average for mean and variance or not.
         fuse_with_relu (bool): if True, this OP performs relu after batch norm.
+        use_global_stats(bool, Default False): Whether to use global mean and
+            variance. In inference or test mode, set use_global_stats to true
+            or is_test to true, and the behavior is equivalent.
+            In train mode, when setting use_global_stats True, the global mean
+            and variance are also used during train time,
+            the BN acts as scaling and shiffting.
```

Comment:
What does this mean: `the BN acts as scaling and shiffting.`

BN always acts as scaling and shifting? How is it related to global stats?